File: BankApplicationAPI/BankApplicationAPI/Models/LoanRepaymentLog.cs
namespace BankApplicationAPI.Models;

public partial class LoanRepaymentLog
{
    public int RepaymentId { get; set; }

    public int? LoanId { get; set; }

    public DateTime? RepaymentDate { get; set; }

    public decimal? RepaymentAmount { get; set; }

    public string? EmployeeId { get; set; }

    public string? RepaymentMethod { get; set; }

    public virtual Employee? Employee { get; set; }

    public virtual LoanApplication? Loan { get; set; }
}

File: BankApp/src/app/Modules/LoanRepaymentLog.ts
import { Employee } from "./Employee";
import { LoanApplication } from "./LoanApplication";

export interface LoanRepaymentLog {
    repaymentId?: number | null;
    loanId?: number | null;
    repaymentDate?: Date | null;
    repaymentAmount?: number | null;
    employeeId?: string | null;
    repaymentMethod?: string | null;
    employee?: Employee | null;
    loan?: LoanApplication | null;
}